dev-libs/gobject-introspection-common/gobject-introspection-common-1.80.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+GNOME_ORG_MODULE="gobject-introspection"
+
+inherit gnome.org
+
+DESCRIPTION="Build infrastructure for GObject Introspection"
+HOMEPAGE="https://wiki.gnome.org/Projects/GObjectIntrospection"
```

Comment:
Now it is https://gi.readthedocs.io/

Reply:
Done in https://github.com/gentoo/gentoo/pull/36834/commits/3e58ece98763be422d5cde5e907ccd94776d35af